content/ew-issue-z200-[2017-03-29].yaml

```diff
@@ -0,0 +1,146 @@
+            -
+                title: "Pro Ember.js: An advanced 2-day training event in Berlin"
+                description: "We'll tackle progressive web apps, modular architecture patterns, performance tuning, authentication & more!"
+                advert: true
+                link: http://ember-workshop.simplabs.com
+                domain: simplabs.com
+
+
+    -
+        heading: "Reading"
+        headlines:
+            -
+                title: "EmberConf 2017 Summary"
+                description: ""
+                link: https://github.com/poteto/emberconf-2017
+            -
+                title: "Rendering 10 Million Items with {{vertical-collection}} and Ember.js"
+                description: ""
+                link: https://build.addepar.com/engineering/rendering-10-million-items-with-vertical-collection/
+            -
+                title: "Lessons learned building a REST API theme for WordPress (with Ember)"
+                description: ""
+                link: https://blog.meetserenity.com/2017/03/lessons-learned-building-rest-api-theme-wordpress/
+            -
+                title: "Ember CLI Team Meeting Minutes - 2017-03-16"
+                description: ""
+                link: https://github.com/emberjs/core-notes/blob/master/ember-cli/2017-03/march-16.md
+            -
+                title: "Ember CLI Team Meeting Minutes - 2017-03-23"
+                description: ""
+                link: https://github.com/emberjs/core-notes/blob/master/ember-cli/2017-03/march-23.md
+
+
+
+    -
+        heading: "Code"
+        headlines:
+            -
+                title: "Ember Bug Widget"
+                description: ""
+                link: https://github.com/elwayman02/ember-bug-widget
+            -
+                title: "Ember CLI Hoverintent"
+                description: ""
+                link: https://github.com/elwayman02/ember-cli-hoverintent
+            -
+                title: "Ember-Tumblr Demo"
+                description: ""
+                link: http://github.jhawk.co/ember-tumblr/
+
+
+    # -
+    #     heading: "Jobs"
+    #     advert: true
+    #     headlines:
+    #         -
+    #            title: ""
+    #            description: ""
+    #            link:
+
+
+    -
+        heading: "Watch/Listen"
+        headlines:
+            -
+                title: "EmberConf 2017 Day 1"
+                description: ""
+                link: https://www.youtube.com/watch?v=6j080toMEKk&t=22602s
+            -
+                title: "EmberConf 2017 Day 2"
+                description: ""
+                link: https://www.youtube.com/watch?v=ZCHFjGSCqP4
+            -
+                title: "Creating an Ember Addon"
+                description: ""
+                link: https://www.youtube.com/watch?v=bhJ6YzBIoWo
+            -
+                title: "Ember.js NYC, March 2017"
+                description: ""
+                link: https://www.youtube.com/watch?v=QV9rfsRZiWM
```

Comment:
This was [included in issue 199](https://github.com/ember-weekly/newsletter/blob/master/content/ew-issue-z199-%5B2017-03-27%5D.yaml#L120). Typically break these meetup videos up into the respective talks where appropriate.